data.json

```diff
@@ -1,4 +1,10 @@
 [
+    {
+    "name": "Aakarsh Teja",
+    "githubUsername": "aakarshteja",
+    "imageUrl": "https://user-images.githubusercontent.com/42816874/92400662-e2e84180-f149-11ea-8304-6e76cab7fb05.png",
+    "issueNumber": 84
+    },
```

Comment:
> in data.json add your own object to the end of the file, your entry should be similar to this example:

Looks good, but can you add it to the end of file please as stated in point `2` in the [instructions](https://github.com/EddieJaoudeCommunity/awesome-github-profiles#to-add-your-github-profile)

Reply:
Updated the code added the data at the end of file.